hostscripts/rpm-packaging/createproject.py

```diff
@@ -82,10 +82,10 @@ def upload_meta(project, build_repository, linkproject):
                   'projectlink': projectlink,
                   'build_repository': build_repository})
 
-    with tempfile.NamedTemporaryFile(delete=False) as meta:
+    with tempfile.NamedTemporaryFile() as meta:
         meta.write(templ)
+        meta.file.flush()
```

Comment:
Yeah, NTF is implementing __getattr__ to delegate any random call to the `file` object. This can work. Doing dir(meta) confuse me because I didn't see other file like methods like `.seek()` or `.flush()`

Done and thanks!